CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # CHANGELOG
 
-## `v67.0.1`
+## `v67.1.0`
```

Comment:
Since `v67.0.1` has already been released, and you can't unrelease it, you should either leave it as-is or release `v67.1.0` separately.

Reply:
@jhendrixMSFT We just merged the code but did not create the tag. It's not a release, right?

Reply:
Oh I see, I thought it had been tagged.  Thanks for clarifying.